pyproject.toml

```diff
@@ -48,8 +48,8 @@ msgpack = "^1.0"
 wsproto = "^1.2"
 h11 = ">=0.13,<0.15"
 # Can use marshmallow or the toasted flavour as you like ;-)
-# marshmallow = "2.14.0"
-toastedmarshmallow = "0.2.6" # updating causes consistency problem in test
+marshmallow = "2.16.2"
+# toastedmarshmallow = "0.2.6" # updating causes consistency problem in test
```

Comment:
Wasn't because `toastedmarshmallow` don't support python-3.11 because it import

```python
from collections import Mapping
```

instead of 

```python
from collections.abc import Mapping
```

?

Reply:
Based on what I've found on internet many people fixed their problem by doing
from collections.abc import Mapping
So I guess that could be the Issue there but I'm not 100% sure about it

Reply:
The stacktrace you've provided seems to indicate that